server.go

```diff
@@ -70,6 +71,12 @@ var (
 	// ErrServerShuttingDown indicates that the server is in the process of
 	// gracefully exiting.
 	ErrServerShuttingDown = errors.New("server is shutting down")
+
+	// Parses a regular expression. The functions in this module let you
```

Comment:
godocs should start with `// validColorRegexp is a regexp that...`

Reply:
@halseth Done